docs/config/README.md

```diff
@@ -24,13 +24,13 @@ disabled = true
 You can change default configuration file location with `STARSHIP_CONFIG` environment variable:
 
 ```sh
-export STARSHIP_CONFIG=~/.starship/config.toml
+export STARSHIP_CONFIG=~/.config/starship.toml
```

Comment:
I am not sure these ones should be changed, this is an example of using the `STARSHIP_CONFIG` environment variable to use a non-default file.

Reply:
Ah, got you. I'll close the pr